doc-surrealist_versioned_docs/version-2.x/tutorials/How-to-define-a-schema-in-Surrealist.mdx

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 2
+sidebar_label: Defining A Schema In Surrealist
+title: Surrealist | Embedding Surrealist  
+description: Surrealist is a user interface for interacting with your SurrealDB database visually. It enables you to seamlessly connect to any SurrealDB instance, allowing you to execute queries, explore your tables, design your schemas, and much more.
+---
+import { SurrealistMini } from "@site/src/components/SurrealistMini"
+
+# Defining A Schema In Surrealist
+
+Surrealist allows you to design your schemas both visually in the [designer view](/docs/surrealist/getting-started/designing-the-database-schema) and with [queries in the query view](/docs/surrealist/getting-started/sending-queries), making it easy to create and manage your tables and relationships. 
+
+In this guide, you will walk through defining a schema in Surrealist.
+
+## What is a schema?
+
+A schema is a blueprint of your database that defines the structure of your tables, the relationships between them, and the constraints that apply to your data. 
+
+## Creating a new schema in the designer view
+
+In Surrealist, you can create a new schema in the designer view by defining your tables and their columns, setting primary and foreign keys, and establishing relationships between your tables.
```

Comment:
Maybe something else instead of "primary and foreign keys" which feels more like SQL terminology. Could replace it with pretty much anything